### PR TITLE
VIDEO-8647 - Fixing Chrome Docker Image

### PR DESCRIPTION
**Pull request by PikaJoyce**
JIRA Ticket : [VIDEO-8647](https://issues.corp.twilio.com/browse/VIDEO-8647)

This PR addresses our Circle CI Job that builds our chrome images. Previously our job was not updating chrome to the latest stable version. We have updated to pulling from `cimg/node` instead of `circleci/node` as that is now deprecated. I've added updates to `beta` and `unstable` chrome Dockerfile as well.

**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### Before review
* [ ] Updated CHANGELOG.md if necessary
* [ ] Added unit tests if necessary
* [ ] Updated affected documentation
* [ ] Verified locally with `npm test`
* [ ] Manually sanity tested running locally
* [ ] Included screenshot as PR comment (if needed)
* [ ] Ready for review

### CODE REVIEW

File: .circleci/images/chrome/stable/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # https://hub.docker.com/r/browserless/chrome/dockerfile/
-FROM circleci/node:lts-browsers
+FROM cimg/node:lts
```
Comment (makarandp0) [inline .circleci/images/chrome/stable/Dockerfile:2]: you are using `cimg/node:current` in other Dockerfile and , `cimg/node:lts` here - why?

**Comment by makarandp0** [comment]
should make same change for firefox images?

File: .circleci/images/chrome/stable/Dockerfile (hunk 1 above)
Comment (PikaJoyce) [reply .circleci/images/chrome/stable/Dockerfile:2]: @makarandp0 The other ones are on beta / unstable and @charliesantos and I were looking at the [image tags](https://circleci.com/developer/images/image/cimg/node#browsers) from circleci and decided that it makes sense for stable to be the current stable [node version](https://nodejs.org/en/) which is `16.14.0`, while "current" would be the latest features `17.6.0`.

File: .circleci/config.yml [hunk 2]
```diff
@@ -294,11 +294,11 @@ workflows:
   Build_Docker_Images_Workflow:
     triggers:
       - schedule:
-          cron: "0 0 * * *"
```
Comment (makarandp0) [inline .circleci/config.yml:297]: revert changes to this file.

**Review by makarandp0 — CHANGES_REQUESTED** [review]
@PikaJoyce, Please revert config.yml changes and update base branch to master.

File: .circleci/images/chrome/beta/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM circleci/node:latest
+FROM cimg/node:current-browsers
```
Comment (charliesantos) [inline .circleci/images/chrome/beta/Dockerfile:1]: Just to double check, what is the version of node in this image?

Comment (PikaJoyce) [reply .circleci/images/chrome/beta/Dockerfile:1]: 17.6.0 is the current version.

**Review by charliesantos — APPROVED** [review]
Please try the integration tests prior merging.

**Review by makarandp0 — APPROVED** [review]

**Pull request merged**